### ISSUE TITLE
CI integration

### ISSUE THREAD

**Issue by MangoIV** [MangoIV/cabal-audit#27]
it would be good to offer a `--ci` option that does sensible things for running in ci, there would have to be a config file to specify advisories that are not applicable, etc.

**Comment by MangoIV** [comment]
Yes, that’s possible! Good idea. For the standard tool I would not do this though.

**Comment by telser** [comment]
> Yes, that’s possible! Good idea. For the standard tool I would not do this though.

Awesome thank you! Some option to fail CI will be great.

**Comment by elland** [comment]
TODO:

- [ ]  Allow ignoring specific advisories for a given package and version range
- [ ] Specify if finding advisories is a warning or an error with non-zero exit codes so CI steps can fail.
- [ ] Define a configuration file schema.

**Comment by NorfairKing** [comment]
I'd like this!

**Comment by MangoIV** [comment]
@elland how is this going, should I take over?
